src/map/script.c

```diff
@@ -4130,6 +4130,14 @@ void script_check_buildin_argtype(struct script_state* st, int func)
 					invalid++;
 				}
 				break;
+			case 'x':
+				if( !data_islabel(data) && !data_isfunclabel(data) && !data_isstring(data) && !( data_isreference(data) && is_string_variable(name) ) ) {
```

Comment:
code style (as described in [chapter 3](https://github.com/HerculesWS/Hercules/wiki/Coding-Style#chapter-3-placing-braces-and-spaces)): this should be instead (see the spaces removed from inside the parentheses, and added between `if` and `(`).

```
if (!data_islabel(data) && !data_isfunclabel(data) && !data_isstring(data) && !(data_isreference(data) && is_string_variable(name))) {
```